src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/NavigationBar.tsx

```diff
@@ -0,0 +1,202 @@
+import * as React from 'react';
+import cn from 'classnames';
+import { Grid, makeStyles, useTheme, useMediaQuery } from '@material-ui/core';
+
+import { Triggers } from 'src/types';
+import { FormLayoutActions } from 'src/features/form/layout/formLayoutSlice';
+import { getTextResource } from '../../utils/formComponentUtils';
+import { useAppDispatch, useAppSelector } from 'src/common/hooks';
+
+const useStyles = makeStyles((theme) => ({
+  menu: {
+    listStyleType: 'none',
+    textDecoration: 'none',
+    paddingLeft: '0px !important', // ".form-group.a-form-group ul" selector is stronger
+    display: 'flex',
+    flexWrap: 'wrap',
+    gap: 6,
+
+    [theme.breakpoints.down(600)]: {
+      flexDirection: 'column',
+    },
+  },
+  containerBase: {
+    borderRadius: '40px',
+
+    '&:active': {
+      backgroundColor: theme.altinnPalette.primary.blueDark,
+    },
+  },
+  buttonBase: {
+    cursor: 'pointer',
+    background: 'none',
+    font: 'inherit',
+    border: `2px solid ${theme.altinnPalette.primary.blueMedium}`,
+    width: '100%',
+    height: '100%',
+    display: 'block',
+    textAlign: 'left',
+    padding: '6px 12px',
+    borderRadius: '40px',
+    fontSize: '1.6rem',
+
+    '&:hover': {
+      outline: `2px solid ${theme.altinnPalette.primary.blueMedium}`,
+    },
+  },
+  buttonSelected: {
+    color: theme.altinnPalette.primary.white,
+    backgroundColor: theme.altinnPalette.primary.blueDarker,
+  },
+  dropdownMenuContent: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  dropdownIcon: {
+    marginLeft: '1rem',
+    marginTop: '0 !important', // "".form-group i" selector is stronger
+    fontSize: '1em !important', // "".form-group i" selector is stronger
+  },
+}));
+
+export interface INavigationBar {
+  triggers?: Triggers[];
+}
+
+interface INavigationButton {
+  onClick: () => void;
+  children: any;
+  current: boolean;
+  hidden?: boolean;
+}
+
+const NavigationButton = React.forwardRef(
+  (
+    { onClick, hidden = false, children, current, ...rest }: INavigationButton,
+    ref: any,
+  ) => {
+    const classes = useStyles();
+
+    return (
+      <button
+        hidden={hidden}
+        type='button'
+        className={cn(classes.buttonBase, {
+          [classes.buttonSelected]: current,
+        })}
+        onClick={onClick}
+        ref={ref}
+        {...(current && { 'aria-current': 'page' })}
+        {...rest}
+      >
+        {children}
+      </button>
+    );
+  },
+);
+
+NavigationButton.displayName = 'Button';
```

Comment:
```suggestion
NavigationButton.displayName = 'NavigationButton';
```
Is `NavigationButton` better here?

Reply:
Absolutely, I think this was left over from some early testing :) Thanks